base/templates/base/userprofile.html

```diff
@@ -0,0 +1,76 @@
+{% extends "base.html" %}
```

Comment:
You have used the base.html and then down there you use the navbar. Choose and work with one.